wave_lang/kernel/ops/wave_ops.py

```diff
@@ -1209,6 +1209,10 @@ def indexing_dims(self) -> list[IndexSymbol]:
     def type(self) -> "Memory":
         return Memory[(*self.shape, self.address_space, self.dtype)]
 
+    @property
+    def get_tile_size(self) -> int:
```

Comment:
This is not very informative name, something like `@property def unpadded_shape()` (which return full shape) will be better.

Reply:
That makes sense, changed it to reflect the final dimension unpadded shape